src/utils/wallet.js

```diff
@@ -590,7 +591,7 @@ class Wallet {
         const balance = await account.getAccountBalance()
 
         // TODO: Should lockup contract balance be retrieved separately only when needed?
-        const re = new RegExp(`\.${ACCOUNT_ID_SUFFIX}$`);
+        const re = new RegExp(`.${ACCOUNT_ID_SUFFIX}$`);
```

Comment:
@mattlockyer why this change? It's not what intended in original regexp (matching a `.`)

Reply:
yarn recommended the escape character as unnecessary

Reply:
![image](https://user-images.githubusercontent.com/321340/93266347-c1e1ab00-f75e-11ea-909c-639df7587054.png)

Reply:
got catch, looks like has to be `new RegExp(`\\.${ACCOUNT_ID_SUFFIX}$`);